.github/workflows/backport.yml

```diff
@@ -1,21 +1,23 @@
 name: Backport
 on:
   pull_request_target:
-    types: [closed, labeled]
+    types: [ closed, labeled ]
 jobs:
   backport:
     name: Backport Pull Request
-    if: github.repository_owner == 'NixOS' && github.event.pull_request.merged == true && (github.event_name != 'labeled' || startsWith('backport', github.event.label.name))
+    if: github.repository_owner == 'NixOS' && github.event.pull_request.merged ==
+      true && (github.event_name != 'labeled' || startsWith('backport',
+      github.event.label.name))
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f # pin@v2
         with:
           # required to find all branches
           fetch-depth: 0
           ref: ${{ github.event.pull_request.head.sha }}
       - name: Create backport PRs
         # should be kept in sync with `version`
```

Comment:
Also needs to be updated below.

Reply:
Which one needs to be updated?

Reply:
https://github.com/NixOS/nixpkgs/blob/040ddaf3b00e7df253f342c1bbc07c5df060c12a/.github/workflows/backport.yml#L23-L24

Reply:
Thanks, got it.